hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/dispatcher/MonoTransactionDispatcherTest.java

```diff
@@ -653,13 +658,88 @@ private static Stream<Arguments> getDispatchParameters() {
                         (Function<TransactionHandlers, TransactionHandler>)
                                 TransactionHandlers::cryptoDeleteLiveHashHandler),
 
+                // file
+                Arguments.of(
+                        TransactionBody.newBuilder()
+                                .fileCreate(FileCreateTransactionBody.DEFAULT)
+                                .build(),
+                        (Function<TransactionHandlers, TransactionHandler>) TransactionHandlers::fileCreateHandler),
+                Arguments.of(
+                        TransactionBody.newBuilder()
+                                .fileUpdate(FileUpdateTransactionBody.DEFAULT)
+                                .build(),
+                        (Function<TransactionHandlers, TransactionHandler>) TransactionHandlers::fileUpdateHandler),
+                Arguments.of(
```

Comment:
I hate that code.
We should extract a lambda that does all the dirty work and have simple lines that creates the arguments:

```
final BiFunction<Function<TransactionBody.Builder, TransactionBody.Builder>, Function<TransactionHandlers, TransactionHandler>, Arguments> argumentSupplier = (transactionBuilderTransformer, handlerFactory) -> {
    final TransactionBody.Builder builder = TransactionBody.newBuilder();
    final TransactionBody body = transactionBuilderTransformer.apply(builder).build();
    return Arguments.of(body, handlerFactory);
};
argumentSupplier.apply(b -> b.cryptoCreateAccount(CryptoCreateTransactionBody.DEFAULT),
        TransactionHandlers::cryptoCreateHandler);
```

Reply:
Good idea, will change it.